_sass/components/share.scss

```diff
@@ -1,33 +1,29 @@
 .share {
   border-top: 1px solid $gray-lighter;
   margin: 0 auto;
-  padding: $md 0 $md 0;
+  padding: $md 0 0 0;
   text-align: center;
 
   @include media(xs) {
     flex-direction: column;
     padding: $md 0;
   }
+  .share-icons {
+    height: 48px;
+    opacity: .75;
+    transition: opacity .125s ease-in-out;
+    width: 48px;
 
-  .btn + .btn {
-    margin-left: $xs;
+    img {
```

Comment:
Also it's preferred if selectors `&` selectors or sibling selectors (`+`,`~`) go before class selectors.